web/src/features/panels/LeftPanel.tsx

```diff
@@ -111,7 +125,10 @@ function OuterPanel({ children }: { children: React.ReactNode }) {
     >
       {isMobile && <MobileHeader />}
       <section className="h-full w-full">{children}</section>
-      <CollapseButton isCollapsed={!isOpen} onCollapse={onCollapse} />
+      <div className="left-full top-2 flex flex-col space-y-20">
```

Comment:
```suggestion
      <div className="left-full top-2 flex flex-col gap-20">
```
This should use gap, the native way to add space between objects in flexboxes.